jlm/llvm/backend/dot/DotWriter.cpp

```diff
@@ -144,9 +144,17 @@ CreateGraphNodes(util::Graph & graph, rvsdg::Region & region, util::Graph * type
     auto & argument = *region.argument(n);
     AttachNodeOutput(node, argument, typeGraph);
 
+    // Give the argument a label using its local index, not the global argument index
+    // Use an uppercase A to distinguish
+    node.SetLabel(util::strfmt("a", n));
+
```

Comment:
It says "uppercase A" in the comment, but then a lowercase a is used.

Reply:
Also to distinguish it from what?

Reply:
Good catch, this comment is not correct